clients/config_client/config_client.go

```diff
@@ -94,12 +94,20 @@ func NewConfigClient(nc nacos_client.INacosClient) (*ConfigClient, error) {
 	if err != nil {
 		return config, err
 	}
-	err = logger.InitLogger(logger.Config{
+	loggerConfig := logger.Config{
 		Level:        clientConfig.LogLevel,
 		OutputPath:   clientConfig.LogDir,
 		RotationTime: clientConfig.RotateTime,
 		MaxAge:       clientConfig.MaxAge,
-	})
+	}
+	if clientConfig.LogSampling != nil {
```

Comment:
When clientConfig.LogSampling != nil.  check`initial, thereafter,tick`,default value provided for ineligible in the SetClientConfig() func